src/proxy/NearCachedMapProxy.ts

```diff
@@ -109,12 +109,12 @@ export class NearCachedMapProxy<K, V> extends MapProxy<K, V> {
     protected deleteInternal(keyData: Data): Promise<void> {
         this.nearCache.invalidate(keyData);
         return super.deleteInternal(keyData)
-            .then<void>(this.invalidateCacheEntryAndReturn.bind(this, keyData));
+            .then<void>(() => this.invalidateCacheEntryAndReturn(keyData));
     }
 
     protected evictInternal(key: Data): Promise<boolean> {
         return super.evictInternal(key)
-            .then<boolean>(this.invalidateCacheEntryAndReturn.bind(this, key));
+            .then<boolean>((bool) => this.invalidateCacheEntryAndReturn(key, bool));
```

Comment:
Minor: We could rename the parameter as `evicted` (instead of `bool`) to make it more readable